ecs-cli/modules/cli/servicediscovery/servicediscovery_app_test.go

```diff
@@ -231,7 +232,7 @@ func TestCreateServiceDiscoveryWithECSParams(t *testing.T) {
 			Description: testDescription,
 		},
 		ServiceDiscoveryService: utils.ServiceDiscoveryService{
-			Name:        testServiceName,
+			Name:        otherTestServiceName,
```

Comment:
is this is only used once, I wouldn't necessary have it as a const. Also I might rename this to "sds-name" or something more descriptive than "other" to specify that it's coming from the SDS params.